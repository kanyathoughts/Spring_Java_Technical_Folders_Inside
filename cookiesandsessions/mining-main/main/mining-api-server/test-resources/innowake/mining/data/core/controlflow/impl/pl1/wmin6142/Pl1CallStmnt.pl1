 root:
  procedure options(Main);
  declare
   proc1 entry (character(15));
  put sKip list('root');
  call proc1('proc1');
 end root;
 proc1:
  procedure(c) ;
  declare
   c character(15);
  put sKip list(c);
 end proc1;