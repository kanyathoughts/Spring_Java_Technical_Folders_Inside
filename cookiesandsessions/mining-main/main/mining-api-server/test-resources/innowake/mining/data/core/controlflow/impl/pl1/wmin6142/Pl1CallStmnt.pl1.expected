 #2 ProcedureBlock, FROM={TestEntryPoint}, TO={#6}
  #6 ProcedureStatement, FROM={#2}, TO={#15}
  #15 DeclareStatement, FROM={#6}, TO={#35}
  #35 PutStatement, FROM={#15}, TO={#50}
  #50 CallStatement, FROM={#35}, TO={#65(CALL)}
  #61 EndStatement, FROM={#108}, TO={TestHaltPoint}
 #65 ProcedureBlock, FROM={#50(CALL), TestEntryPoint}, TO={#69}
  #69 ProcedureStatement, FROM={#65}, TO={#77}
  #77 DeclareStatement, FROM={#69}, TO={#92}
  #92 PutStatement, FROM={#77}, TO={#108}
  #108 EndStatement, FROM={#92}, TO={#61, TestReturnPoint}
